### PR TITLE
chore(compliance): implements dynamic handling of available compliance frameworks

### PR DESCRIPTION
**Pull request by pedrooot**
### Context

Implements a method to handle compliance names in a dynamic way

### Description

Replaces the compliance list with a loop to take the names from compliance dir

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/config/config.py [hunk 1]
```diff
@@ -18,28 +18,17 @@
...
-]
+compliance_aws_dir = "./prowler/compliance/aws"
+available_compliance_frameworks = []
+with os.scandir(compliance_aws_dir) as files:
+    files = [
+        file.name
+        for file in files
+        if file.is_file()
+        and file.name.endswith(".json")
+        and available_compliance_frameworks.append(file.name.split(".")[0])
```
Comment (MrCloudSec) [inline prowler/config/config.py]: Have you though about the case that a framework name contains a `.` (for example cis_1.4_aws)?

**Review by MrCloudSec — CHANGES_REQUESTED** [review]
Awesome job!!
Can you please remove the Pipfile files?
Also, check my comment.

File: prowler/config/config.py (hunk 1 above)
Comment (pedrooot) [reply prowler/config/config.py]: True!! I fixed it, thanks Sergio

File: prowler/config/config.py [hunk 2]
```diff
@@ -18,28 +18,17 @@
...
-]
+compliance_aws_dir = "./prowler/compliance/aws"
+available_compliance_frameworks = []
+with os.scandir(compliance_aws_dir) as files:
+    files = [
+        file.name
+        for file in files
+        if file.is_file()
+        and file.name.endswith(".json")
+        and available_compliance_frameworks.append(file.name.split(".j")[0])
```
Comment (JeffreySouza) [inline prowler/config/config.py]:
```suggestion
        and available_compliance_frameworks.append(file.name.removesuffix('.json')
```

Comment (pedrooot) [reply prowler/config/config.py]: Better! Thanks @JeffreySouza

**Review by MrCloudSec — APPROVED** [review]

**Pull request merged**